src/components/molecules/DataResultItem.tsx

```diff
@@ -1,15 +1,22 @@
 import React from 'react';
-import { Card, Pane, Heading, Text } from 'evergreen-ui';
+import { Card, Heading, Pane, Text } from 'evergreen-ui';
 import DatasetInfoBox from '../atoms/DatasetInfoBox';
+import { ParentSize } from '@visx/responsive';
+import ThresholdChart from '../charts/ThresholdChart';
+import LineChart from '../charts/LineChart';
+import mockTimeEntry from '../../mockdata/mockTimeEntry';
+import { VisualisationType } from '../../types/Metadata';
 
 export type DataResultItemProps = {
-    visualisationType: string;
+    visualisationType: VisualisationType;
     title: string;
     description: string;
     tags: string[];
 };
 
 const DataResultItem: React.FC<DataResultItemProps> = ({ title, description, tags, visualisationType }) => {
+    const timeEntryMockData = mockTimeEntry(100, undefined);
```

Comment:
What is the second param?

Reply:
Minmax-tuple, defaults to [0, 100]. Could be changed, but felt it was decent as is.

Reply:
That is very very true.